.travis.yml

```diff
@@ -8,6 +8,7 @@ rvm:
 - 2.5.0
 - 2.6.0
 - 2.7.0
+cache: bundler
```

Comment:
Hi @kikihakiem, thanks for adding this, did not know about this option!

Reply:
But since this project's dependencies are small, it only saves ~2 seconds when executing bundle install 😅